src/BenchmarksServer/applicationHost.config

```diff
@@ -0,0 +1,733 @@
+    </providers>
+  </configProtectedData>
+
+  <system.applicationHost>
+
+    <applicationPools>
+      <add name="DefaultAppPool" />
+      <applicationPoolDefaults managedRuntimeVersion="v4.0">
+        <processModel identityType="ApplicationPoolIdentity" />
+      </applicationPoolDefaults>
+    </applicationPools>
+
+    <!--
+
+      The <customMetadata> section is used internally by the Admin Base Objects
+      (ABO) Compatibility component. Please do not modify its content.
+
+    -->
+    <customMetadata />
+
+    <!--
+
+      The <listenerAdapters> section defines the protocols with which the
+      Windows Process Activation Service (WAS) binds.
+
+    -->
+    <listenerAdapters>
+      <add name="http" />
+    </listenerAdapters>
+
+    <log>
+      <centralBinaryLogFile enabled="true" directory="%SystemDrive%\inetpub\logs\LogFiles" />
+      <centralW3CLogFile enabled="true" directory="%SystemDrive%\inetpub\logs\LogFiles" />
+    </log>
+
+    <sites>
+      <site name="Default Web Site" id="1">
+        <application path="/">
+          <virtualDirectory path="/" physicalPath="c:\Users\pakrymet\AppData\Local\Temp\tmpC1E3.tmp\hwgaw5ql.4g3\benchmarks\" />
+        </application>
+        <bindings>
+          <binding protocol="http" bindingInformation="*:8009:localhost" />
+        </bindings>
+      </site>
+      <siteDefaults>
+        <logFile logFormat="W3C" directory="%SystemDrive%\inetpub\logs\LogFiles" />
+        <traceFailedRequestsLogging directory="%SystemDrive%\inetpub\logs\FailedReqLogFiles" />
+      </siteDefaults>
+      <applicationDefaults applicationPool="DefaultAppPool" />
+      <virtualDirectoryDefaults allowSubDirConfig="true" />
+    </sites>
+
+    <webLimits />
+
+  </system.applicationHost>
+
+  <system.webServer>
+
+    <asp />
+
+    <caching enabled="true" enableKernelCache="true">
+    </caching>
+
+    <cgi />
+
+    <defaultDocument enabled="true">
+      <files>
+        <add value="Default.htm" />
+        <add value="Default.asp" />
+        <add value="index.htm" />
+        <add value="index.html" />
+        <add value="iisstart.htm" />
+      </files>
+    </defaultDocument>
+
+    <directoryBrowse enabled="false" />
+
+    <fastCgi />
+
+    <!--
+
+      The <globalModules> section defines all native-code modules.
+      To enable a module, specify it in the <modules> section.
+
+    -->
+    <globalModules>
```

Comment:
It's worth considering whether we should remove all modules besides ANCM (and I guess Anonymous auth which is required)?

Reply:
I think that's reasonable for benchmarks, as I know many folks do that in their own apps today. For the stress (long running) scenarios, we could add them back, or have two versions to help get better coverage of different environments.

Reply:
@shirhatti do you have the minimal list of required modules? I tried removing all but `AspNetCore` and got an empty page. 

I think I'll merge fist iteration as is file an issue to reduce the module count.